### ISSUE TITLE
Failing contrast ratio for multiple elements in main app page

### ISSUE THREAD

**Issue by CitrusFruits** [tootsuite/mastodon#11420]
There are several places in the Mastodon web application (v2.9.2) that have failing contrast ratio according to WCAG AA guidelines. I haven't checked everywhere in the app, but here are places I've found upon a first high level look. For reference, a ratio of 4.5 is considered passing.

**Hash tags and @ mentions**
*Contrast ratio: 3.45*
![hash_tags_and_at_mentions](https://user-images.githubusercontent.com/2839268/62000134-deb3ae80-b084-11e9-8e53-0feb6b61718e.jpg)
**Search placeholder text**
*Contrast ratio: 3.49*
**Search icon**
*Contrast ratio: 1.75*
![search](https://user-images.githubusercontent.com/2839268/62000133-dce9eb00-b084-11e9-8e8b-13030e45669b.jpg)
**Toot button**
*Contrast ratio: 3.45*
![toot_button](https://user-images.githubusercontent.com/2839268/62000135-e07d7200-b084-11e9-9ebe-bb06f9d57eaa.png)

Obviously, these are all on the light theme, but looking around the dark theme I think there's also some opportunities for improvement.

I hope this is the right place to put this. If there's interest, I could do a more comprehensive audit to get a better scope of all the places this happens and then we could go from there. I'm also capable of creating a PR, but would like to wait on that until I get buy in and UX decisions are agreed upon.

**Comment by mjankowski** [comment]
> I'm also capable of creating a PR, but would like to wait on that until I get buy in and UX decisions are agreed upon.

Bumping this one ... despite this being five years old, whether THIS person is available or not, I suspect we could absolutely find some WCAG-expert contributors to chip away at stuff like this if there was vague desire from internal design/UX to either collaborate or review?

**Comment by CitrusFruits** [comment]
I'm still around! Although I have less bandwidth than I used to. I might be able to help with a high level UX audit still as a starting point, but would love the buy-in of a maintainer (or whomever has rights to approve PRs) before I got through the trouble.

**Comment by diondiondion** [comment]
This has been addressed in Mastodon 4.6
